modin/engines/base/io/column_stores/column_store_dispatcher.py

```diff
@@ -40,6 +64,26 @@ def call_deploy(cls, fname, col_partitions, **kwargs):
 
     @classmethod
     def build_partition(cls, partition_ids, row_lengths, column_widths):
+        """Build array with partitions of `cls.frame_partition_cls` class
+        from data ids passed as `partition_ids` and partitions metadata
+        passed as `row_lengths` and `column_widths`.
+
+        Parameters
+        ----------
+        partition_ids: list
+                array with references to the partitions data.
+        row_lengths: list
+                Partitions rows lengths.
+        column_widths: list
+                Number of columns in each partition.
+
+        Returns
+        -------
+        np.array:
+            array with shape equals to the shape of `partition_ids` and
+            filed with partitions objects.
```

Comment:
```suggestion
            filed with partition objects.
```
(assuming each array cell contains one partition you should use plural form only for the container part of the sentence)